carbon-design-system/carbon-addons-iot-react
# docs(table): create story and mdx for states

Closes #3333

**Summary**

Create story and docs

**Change List (commits, features, bugs, etc)**
- new story
- updated playground
- new mdx

**Acceptance Test (how to verify the PR)**

- Test the story at http://localhost:3000/?path=/docs/1-watson-iot-table--with-main-view-states
- Try the related docs
- Try the playground tab "Main view states"

**Regression Test (how to make sure this PR doesn't break old functionality)**
- 



**Things to look for during review**
